api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/instance/InstanceRefreshService.java

```diff
@@ -149,16 +158,16 @@ private void processServiceInstance(Set<String> containersUpdated, Applications
     }
 
     /**
-     * Go ahead amd retrieve this instances API doc and update the cache
+     * Go ahead and retrieve this instances API doc and update the cache
```

Comment:
This doesn't make much sense grammatically. 
"Ensure that you retrieve the API doc of this instance, and update the cache". Also we need to make sure it's understood what instance we refer to here. What is "This instance"?